core/lib/eth_client/src/clients/mock.rs

```diff
@@ -591,10 +575,25 @@ mod tests {
     use super::*;
     use crate::{CallFunctionArgs, EthInterface};
 
+    macro_rules! base_fees {
```

Comment:
Nit: Why can't this be a function? Functions are more maintainable than macros.

Reply:
Personal preference, IMHO short macros in tests are perfectly readable.
No strong opinion, will change.